src/.jenkins/test/docker-compose.yml

```diff
@@ -1,9 +1,9 @@
-version: "3.0"
+version: "3.4"
```

Comment:
nitpick: according to this: https://docs.docker.com/compose/compose-file/compose-file-v3/
And the readme recommending Docker CE >= 18.09 we can use 3.7 here.

Originally I upgraded other services from 3.0 to 3.4, which was the lowest version which supported some feature i needed in compose files.